slave-gradle/Dockerfile.rhel7

```diff
@@ -0,0 +1,44 @@
+FROM openshift/jenkins-slave-base-rhel7
+
+# Labels consumed by Red Hat build service
+LABEL com.redhat.component="jenkins-slave-gradle-rhel7-docker" \
+      name="openshift3/jenkins-slave-gradle-rhel7" \
+      version="3.6" \
+      architecture="x86_64" \
+      release="4" \
+      io.k8s.display-name="Jenkins Slave Gradle" \
+      io.k8s.description="The jenkins slave gradle image has the gradle tools on top of the jenkins slave base image." \
+      io.openshift.tags="openshift,jenkins,slave,gradle"
+
+ENV GRADLE_VERSION=3.4.1 \
+    BASH_ENV=/usr/local/bin/scl_enable \
+    ENV=/usr/local/bin/scl_enable \
+    PROMPT_COMMAND=". /usr/local/bin/scl_enable"
+ENV PATH=$PATH:/opt/gradle-${GRADLE_VERSION}/bin
+
+# Install Maven
+RUN yum-config-manager --enable rhel-server-rhscl-7-rpms && \    
+    yum-config-manager --disable epel >/dev/null || : && \
+    INSTALL_PKGS="java-1.8.0-openjdk-devel" && \
+    yum install -y $INSTALL_PKGS && \
+    curl -LOk https://services.gradle.org/distributions/gradle-${GRADLE_VERSION}-bin.zip && \
+    unzip gradle-${GRADLE_VERSION}-bin.zip -d /opt && \
+    rm -f gradle-${GRADLE_VERSION}-bin.zip && \
+    rpm -e java-1.7.0-openjdk --nodeps || true && \
+    rpm -e java-1.7.0-openjdk-devel --nodeps || true && \
+    rpm -e java-1.7.0-openjdk-headless --nodeps || true && \
```

Comment:
these lines were only needed in the maven image because the maven package mistakenly drags these deps in.  you don't need them here.  (unless the gradle package also drags jdk1.7)